lib/yoti/sandbox/const.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+module Sandbox
+  SANDBOX_BASE_URL = 'https://dev1.api.internal.yoti.com/sandbox'
+  SANDBOX_ENDPOINT = '/apps'
+end
```

Comment:
Do we really want an internal URL inside of the public SDK?  This should probably be kept in the sdk integration test repo and passed to the sandbox client in a constructor.